ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt

```diff
@@ -312,39 +318,39 @@ class MainService : Service() {
         return port
     }
 
-    suspend fun loadModels() {
+    fun loadModels() {
```

Comment:
This looks like a significant change. Please describe the details in the commit body.

Reply:
Updated it
Thank you :)